.github/CODEOWNERS

```
@@ -44,8 +44,15 @@ docker-compose-docs.yml                                 @DataDog/corpweb
 # Cloud Cost Management
 documentation/content/en/cloud_cost_management/_index.md         @DataDog/cloud-cost-management-visibility @DataDog/documentation
 
+# Log collection
+content/en/logs/log_collection/ios/                            @Datadog/rum-mobile @DataDog/documentation
+
+# Traces
+content/en/tracing/trace_collection/dd_libraries/ios.md
```

You may also want to add them here:
```suggestion
content/en/tracing/trace_collection/dd_libraries/ios.md
```